tools/roslaunch/src/roslaunch/loader.py

```diff
@@ -491,7 +491,44 @@ def param_value(self, verbose, name, ptype, value, textfile, binfile, command):
                 print("... executing command param [%s]" % command)
             import subprocess, shlex #shlex rocks
             try:
-                p = subprocess.Popen(shlex.split(command), stdout=subprocess.PIPE)
+                if os.name == 'nt':
+                    # Python scripts in ROS tend to omit .py extension since they could become executable with shebang line
+                    # special handle the use of Python scripts in Windows environment:
+                    # 1. search for a wrapper executable (of the same name) under the same directory with stat.S_IXUSR flag
+                    # 2. if no wrapper is present, prepend command with 'python' executable
+
+                    cl = shlex.split(command, posix=False) # use non-posix method on Windows
```

Comment:
Nitpick: PEP8 - two spaces before the `#`.

Reply:
thanks for sharing the coding style to follow!